### PR TITLE
Update the Standard-Bridge-Standard-Token tutorial of OVM 2.0

### PR DESCRIPTION
**Pull request by qbzzt**
Draft until I write a detailed description

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-*       @smartcontracts
+*       @K-Ho
```
Comment (elenadimitrova) [inline .github/CODEOWNERS:1]: This should be done in a separate PR so it can be merged quicker. Also if @K-Ho is a code owner here, he should be assigned as the reviewer of the PRs. @K-Ho do you have the capacity to be reviewing all tutorials?

Comment (qbzzt) [reply .github/CODEOWNERS:1]: OK, I created a separate PR just for this change. Here is the URL: https://github.com/ethereum-optimism/optimism-tutorial/pull/72

**Comment by qbzzt** [comment]
It will be easier to just make a separate PR
